pipelines/integration_resolver_pipeline_fail.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: integration-resolver-pipeline-pass
+spec:
+  tasks:
+    - name: task-success
+      taskRef:
+        resolver: git
+        params:
+          - name: org
+            value: redhat-appstuio
+          - name: repo
+            value: integration-examples
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: tasks/test_output.yaml
+      params:
+        - name: RESULT
+          value: SUCCESS
+    - name: task-fail
+      taskRef:
+        resolver: git
+        params:
+          - name: org
+            value: redhat-appstudio
+          - name: repo
+            value: integration-examples
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: tasks/test_output.yaml
+      params:
+        - name: RESULT
+          value: FAILURE
+    - name: task-skipped
+      taskRef:
+        resolver: git
+        params:
+          - name: org
```

Comment:
Will this `org` be used? I found it is not defined in some CRs.

Reply:
The git resolver can either use `org` and `repo`  or it can specify an `url`. 
Both approaches are valid, but can't be supplied at the same time ([docs](https://tekton.dev/docs/pipelines/git-resolver/)).

Reply:
I see now. Thank you!